go/pkg/cas/upload_test.go

```diff
@@ -354,6 +356,69 @@ func TestSmallBlobs(t *testing.T) {
 	}
 }
 
+func TestStreaming(t *testing.T) {
```

Comment:
FYI if you leave the retries here, we should have tests with retries.

Reply:
`fakes.CAS` currently doesn't support transient errors. Adding it there seems like a large feature (ideally done in all RPCs), so I've added a TODO in this CL